partitura/utils/music.py

```diff
@@ -726,60 +828,65 @@ def compute_pianoroll(note_info, time_unit='auto', time_div='auto',
+            '{0} or "auto", but was given '
+            "{1}".format(", ".join(TIME_UNITS), time_unit)
+        )
+    if time_unit == "auto":
         onset_unit, duration_unit = get_time_units_from_note_array(note_array)
 
     else:
-        onset_unit = f'onset_{time_unit}'
-        duration_unit = f'duration_{time_unit}'
+        onset_unit = f"onset_{time_unit}"
+        duration_unit = f"duration_{time_unit}"
 
-    if time_div == 'auto':
-        if onset_unit in ('onset_beat',
-                          'onset_quarter',
-                          'onset_sec'):
+    if time_div == "auto":
+        if onset_unit in ("onset_beat", "onset_quarter", "onset_sec"):
             time_div = 8
-        elif onset_unit == 'onset_div':
+        elif onset_unit == "onset_div":
             time_div = 1
     else:
         time_div = int(time_div)
 
-    if 'channel' in note_array.dtype.names and remove_drums:
-        LOGGER.info('Do not consider drum track for computing piano roll')
-        non_drum_idxs = np.where(note_array['channel'] != 9)[0]
+    if "channel" in note_array.dtype.names and remove_drums:
+        LOGGER.info("Do not consider drum track for computing piano roll")
+        non_drum_idxs = np.where(note_array["channel"] != 9)[0]
         note_array = note_array[non_drum_idxs]
 
-    piano_roll_fields = ['pitch',
-                         onset_unit,
-                         duration_unit]
-
-    if 'velocity' in note_array.dtype.names:
-        piano_roll_fields += ['velocity']
-
-    pr_input = np.column_stack([note_array[field].astype(float)
-                                for field in piano_roll_fields])
-
-    return _make_pianoroll(note_info=pr_input,
-                           time_div=time_div,
-                           onset_only=onset_only,
-                           note_separation=note_separation,
-                           pitch_margin=pitch_margin,
-                           time_margin=time_margin,
-                           return_idxs=return_idxs,
-                           piano_range=piano_range)
-
-
-def _make_pianoroll(note_info, onset_only=False,
-                    pitch_margin=-1, time_margin=0,
-                    time_div=8, note_separation=True,
-                    return_idxs=False,
-                    piano_range=False,
-                    remove_silence=True,
-                    min_time=None):
+    piano_roll_fields = ["pitch", onset_unit, duration_unit]
+
+    if "velocity" in note_array.dtype.names:
+        piano_roll_fields += ["velocity"]
+
+    pr_input = np.column_stack(
+        [note_array[field].astype(float) for field in piano_roll_fields]
+    )
+
+    return _make_pianoroll(
+        note_info=pr_input,
+        time_div=time_div,
+        onset_only=onset_only,
+        note_separation=note_separation,
+        pitch_margin=pitch_margin,
+        time_margin=time_margin,
+        return_idxs=return_idxs,
+        piano_range=piano_range,
+    )
+
+
+def _make_pianoroll(
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.